# Command arm end-effector xyz/pose

float32 x
float32 y
float32 z
float32 pitch
float32 roll
bool plan
bool wait
bool check_obstruction
---
bool result
